Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -350,7 +348,8 @@ private func createResolvedPackages(
                     // the product we found above.
                     guard let dependencyPackage = packageMapByName[packageName], dependencyPackage.products.contains(product) else {
                         let error = PackageGraphError.productDependencyIncorrectPackage(
-                            name: productRef.name, package: packageName)
+                            name: productRef.name, package: packageName
+                        )
```

Comment:
Are all these formatting changes due to mechanical formatting?  It's very hard to spot semantic changes with all the formatting changes interspersed.

Also, this one looks particularly weird.  Is the closing parenthesis of a function call intended to be on a line by itself?

Reply:
@abertelrud Sorry for the inconvenience. These were automatically added by [SwiftFormat](https://github.com/nicklockwood/SwiftFormat) according to the rules added by #3012. See also https://github.com/apple/swift-package-manager/pull/3012#discussion_r518025194. 

That said, I'm also suspicious of this change, now that you mention it. I'll try consolidating it and see if it sticks.

Reply:
Fixed by d1054e2f.